### ISSUE TITLE
Updating Newtonsoft.Json to version 10

### ISSUE THREAD

**Issue by shoang-pfcu** [docusign/docusign-csharp-client#134]
Whenver I update Newtonsoft.Json to version 10, I keep getting the following error when running DocuSign.eSign.dll
```

The type initializer for 'DocuSign.eSign.Client.Configuration' threw an exception. Could not load file or assembly 'Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The system cannot find the file specified.
```

This is keeping me from updating other Nuget packages that I have.

Have you guys trying building this with newtonsoft.json version 10?

**Comment by brbeaird** [comment]
Believe it or not, I'm still running version 5, lol. But I had to do some unusual steps to get that to work.

Basically, I'm pretty sure the DocuSign NuGet dependence is hard-coded for Newtonsoft version 8, so if you want anything else, you need to build the whole thing from source yourself.

**Comment by joshjeppson** [comment]
Running DocuSign v2.1.0 and Newtonsoft.Json v10.0.3 here. Try adding an entry like the following in your app.config or web.config:

`<runtime><assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1"><dependentAssembly><assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" /><bindingRedirect oldVersion="0.0.0.0-10.0.0.0" newVersion="10.0.0.0" /></dependentAssembly></dependentAssembly></assemblyBinding></runtime>`

**Comment by gsnavin** [comment]
@shoang-pfcu @brbeaird You should be able to use any version greater than 8.0.3. Let me know if this is still an issue.

Between we have published a new version 2.1.7, feel free to try it out with this version.
